src/StubExecutable/StubExecutable.cpp

```diff
@@ -90,11 +163,21 @@ int APIENTRY wWinMain(_In_ HINSTANCE hInstance,
                      _In_ LPWSTR    lpCmdLine,
                      _In_ int       nCmdShow)
 {
-	std::wstring appName;
-	appName.assign(FindOwnExecutableName());
-
-	std::wstring workingDir(FindLatestAppDir());
-	std::wstring fullPath(workingDir + L"\\" + appName);
+	std::wstring fullPath;
+	std::wstring workingDir;
+	// Brave Software specific logic
+	std::wstring braveCore(FindBraveCoreInstall());
```

Comment:
Should we read a runtime parameter flag here so that we can have a shortcut installed on the system which always runs the old Muon browser, as keeping access to that was a requirement AFAIK.

Reply:
That's not a bad idea- we could even then add a new shortcut called "Brave (old)" to the desktop.  However, I think adding that extra complexity might not be worth it since we can direct folks to `%USERPROFILE%\AppData\Local\brave\app-0.25.2` as needed when approached for support